.github/workflows/test.yml

```diff
@@ -33,8 +33,8 @@ jobs:
 
       - name: Install cmake and cmake-format
         run: |
-          pip3 install --upgrade pip
-          pip3 install cmake pyyaml cmake-format
+          sudo pip3 install --upgrade pip
```

Comment:
for whatever reason, the intel runner might not have pip3 installed. i suggest installing it with `apt` in the `Install build tools` step. The package is `python3-pip`